ocs_ci/ocs/utils.py

```diff
@@ -627,9 +627,14 @@ def get_pod_name_by_pattern(pattern='client', namespace='openshift-storage'):
 
 def setup_ceph_toolbox():
     """
-    Setup ceph-toolbox based
+    Setup ceph-toolbox - also checks if toolbox exists, if it exists it
+    behaves as noop.
     """
     namespace = ocsci_config.ENV_DATA['cluster_namespace']
+    ceph_toolbox = get_pod_name_by_pattern('rook-ceph-tools', namespace)
+    if len(ceph_toolbox) == 1:
```

Comment:
Could we ever have more than one toolbox, even if it's unintentional? If so we might want to change `==` to `>=`.

Reply:
Its rare but possible with yaml manipulation outside CI, so I will fix that.

Reply:
I think its better to discourage the ceph-tool-pod yaml manipulation and there is not great advantage in doing that, I am going to dismiss this for now, but will revisit if really required.